<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>601</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>651</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QGridLayout" name="singleValuesLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="aquariumWeightValue">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="aquariumLengthLabel">
        <property name="text">
         <string>Długość akwarium:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="aquariumLengthValue">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="aquariumWeightLabel">
        <property name="text">
         <string>Szerokość akwarium:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>Czas symulacji [s]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="durationValue">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="snailsLabel">
      <property name="text">
       <string>Początkowy stan ślimaków:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="snailsLayout">
      <item>
       <widget class="QTableWidget" name="snailsTable"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="snailsButtonsLayout" stretch="5,5">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addSnailButton">
          <property name="text">
           <string>Dodaj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeSnailButton">
          <property name="text">
           <string>Usuń</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="plantsLabel">
      <property name="text">
       <string>Początkowy stan roślin:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pllantsLayout">
      <item>
       <widget class="QTableWidget" name="plantsTable"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="plantsButtonsLayout" stretch="5,5">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addPlantButton">
          <property name="text">
           <string>Dodaj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePlantButton">
          <property name="text">
           <string>Usuń</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
